[Node.js] Style guide fixes for TDBX vale rules (#842)
mongodb/docs-node: source/fundamentals/logging.txt

Temporary Alternative
---------------------

> The Node.js team is working on rewriting the logger. In the
> meantime, you can output monitor events using the following snippet:

Reviewer: ```suggestion
We are developing a new logging framework. In the
meantime, you can output monitor events using the following snippet:
```

Temporary Alternative
---------------------

> We are developing a new logging framework. In the meantime, you can output monitor events
> using the following snippet:

Reviewer: ```suggestion
We are developing a new logging framework. In the meantime, you can output monitor events
by using the following snippet:
```

Reviewer: https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u:~:text=word%2C%20username.-,using,-by%20using%20with
